odl/tomo/backends/astra_cuda.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Backen for ASTRA using CUDA"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import numpy as np
+
+try:
+    import astra
+    if astra.astra.use_cuda():
+        ASTRA_CUDA_AVAILABLE = True
+except ImportError:
```

Comment:
I got it wrong in my comment. `ASTRA_CUDA_AVAILABLE` should be defined in any case, so even in the `except` clause, we need to define the constant to `False`.